tests/frontDesk/clinicalExamination.js

```diff
@@ -42,35 +42,46 @@ step("put medications <prescriptionNames>", async function (prescriptionNames) {
 })
 
 step("Doctor prescribes medicines <prescriptionNames>", async function (prescriptionNames) {
-    var prescriptionFile = `./bahmni-e2e-common-flows/data/${prescriptionNames}.json`;
-    gauge.dataStore.scenarioStore.put("prescriptions", prescriptionFile)
-    var drugName = gauge.dataStore.scenarioStore.get("Drug Name")
-
-    var medicalPrescriptions = JSON.parse(fileExtension.parseContent(prescriptionFile))
-    gauge.message(medicalPrescriptions)
-
-    if (medicalPrescriptions.drug_name != null) {
-        if (drugName == null)
-            drugName = medicalPrescriptions.drug_name;
-        if (await textBox(toRightOf("Drug Name")).exists()) {
-            await write(drugName, into(textBox(toRightOf("Drug Name"))));
-            await click(link(drugName, below(textBox(toRightOf("Drug Name")))));
-            await dropDown(toRightOf("Units")).select(medicalPrescriptions.units);
-            await dropDown(toRightOf("Frequency")).select(medicalPrescriptions.frequency)
-            await write(medicalPrescriptions.dose, into(textBox(toRightOf("Dose"))));
-            await write(medicalPrescriptions.duration, into(textBox(toRightOf("Duration"))));
-        }
-        else {
-            await write(drugName, into(textBox(below("Drug Name"))));
-            await click(link(drugName, below("Drug Name")));
-            await dropDown(below("Units")).select(medicalPrescriptions.units);
-            await dropDown(below("Frequency")).select(medicalPrescriptions.frequency)
-            await write(medicalPrescriptions.dose, into(textBox(below("Dose"))));
-            await write(medicalPrescriptions.duration, into(textBox(below("Duration"))));
+    if (prescriptionNames.includes(',')) { var prescriptionsList = prescriptionNames.split(',') }
+    else { var prescriptionsList = prescriptionNames.split() }
+    console.log(prescriptionsList)
+    for (var i = 0; i < prescriptionsList.length; i++) {
+
+        var prescriptionFile = `./bahmni-e2e-common-flows/data/${prescriptionsList[i]}.json`;
+        gauge.dataStore.scenarioStore.put("prescriptions", prescriptionFile)
```

Comment:
if we are sending more than one file then only the last executed file will be stored on "prescriptions", we need to parameterize this like the below.
`gauge.dataStore.scenarioStore.put("prescriptions"+i, prescriptionFile)`

And one more line before for loop
`gauge.dataStore.scenarioStore.put("prescriptionsCount", prescriptionsList.length)`

Reply:
done